### PR TITLE
Listing the screencasts we need for part 2

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: cd-screencasts.txt [hunk 1]
```diff
@@ -6,32 +6,36 @@ Each screencasts is meant to be less than a minute long.
...
-- How to add an exercise manually (do we want to cover copy/pasting Markdown?)
+  - Also show how to save work at this point
+  - And mention the "sync with GitHub" option
+- How to add an exercise manually
+  - We have to cover copying and pasting Markdown because the current Teach editor doesn't support all exercise types
+  - Where do authors find blank Markdown templates for exercise types?
 - How to edit an exercise
 - How to preview an exercise
-- How to add a chapter
+- How to add a new chapter
```
Comment (becca-datacamp) [inline cd-screencasts.txt]: How to delete a chapter

Comment (becca-datacamp) [inline cd-screencasts.txt]: How to delete a chapter

File: cd-screencasts.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Short Screencasts to Introduce Content Development Topics
+
+(and what does it say that I wrote "convent development" twice?)
+
+Each screencasts is meant to be less than a minute long.
```
Comment (richierocks) [inline cd-screencasts.txt]: typo

File: cd-screencasts.txt [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - Build "tab"
+  - Settings "tab"
+  - "Build Manually" button
+  - "Refresh" link
+  - "Edit" button (end with this to segue into the next video)
+- What's where in the course editor
+  - Overview pane
+  - Edit/Preview pane
+  - Save
+  - Sync with GitHub
```
Comment (richierocks) [inline cd-screencasts.txt]: Save and sync ought to be a single screencast, in which you emphasize that you need to be careful not to click sync when you meant save, thus overwriting all your hard work.

**Pull request merged**